lib/blockchain.go

```diff
@@ -1465,7 +1465,7 @@ func _FindCommonAncestor(node1 *BlockNode, node2 *BlockNode) *BlockNode {
 	return node1
 }
 
-func CheckTransactionSanity(txn *MsgDeSoTxn, blockHeight uint32, params *DeSoParams) error {
+func CheckTransactionSanity(txn *MsgDeSoTxn, blockHeight uint64, params *DeSoParams) error {
```

Comment:
Rather than maving the uint32 to uint64 switch here, I'd suggestion adding a TODO a making the switch in a separate PR. BlockHeight is passed around as a uint32 in many places, so it will be less error prone if we do the switch in one pass holistically across the codebase where needed.

Reply:
Good point. This change is a leftover from splitting my mempool code lol. I will remove the change from this PR.